packages/tasks/src/pipelines.ts

```diff
@@ -652,6 +652,11 @@ export const PIPELINE_DATA = {
 		modality: "cv",
 		color: "indigo",
 	},
+	"video-text-to-text": {
```

Comment:
There are no models nor datasets with this tag yet (see https://huggingface.co/models?other=video-text-to-text&sort=trending for example). I would add `hideInModels` and `hideInDatasets` to avoid having big buttons not leading to any repos

Reply:
there's a lot of models and some datasets too, I opened PRs to a lot of them

Reply:
btw does this tag apply for video captioning datasets?

Reply:
@zucchini-nlp if they have to input "caption this video"-like prompt yes, if not, we need to have a separate task called `video-to-text` (e.g. like difference between blip and llava)

Reply:
ah, i see, thanks!

Reply:
Yes for pure captioning we'd need the separate `video-to-text` task (similar to `image-to-text`)

Reply:
@osanseviero some PRs are merged, should I merge this?

Reply:
I would still `hideInDatasets`